coordinator/persistence-api/src/main/java/io/stargate/db/schema/ParameterizedType.java

```diff
@@ -157,7 +157,7 @@ public Column.ColumnType fieldType(String name) {
     @Value.Auxiliary
     public int schemaHashCode() {
       return SchemaHashable.combine(
-          SchemaHashable.hashCode(isFrozen()), SchemaHashable.hashCode(parameters()));
+          SchemaHashable.hashCode(isFrozen()), SchemaHashable.hash(parameters()));
```

Comment:
This is important! Former lead to `list.hashCode()` calculation; latter calculates over `List<SchemaHashable>`.
One reason why it probably has not a great idea to provide general `SchemaHashable.hashCode(Object)` method in the first place....